packages/base/src/util/PopupUtils.ts

```diff
@@ -59,13 +59,13 @@ const isClickInRect = (event: MouseEvent | TouchEvent, rect: DOMRect) => {
 	let x;
 	let y;
 
-	if (event instanceof TouchEvent) {
+	if (event instanceof MouseEvent) {
```

Comment:
`TouchEvent` is undefined in Firefox desktop. All other browsers mock its existence, even if they don't have a touch screen.